corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py

```diff
@@ -509,6 +509,30 @@ def _test(self, action_tuples):
         for index, action in action_tuples:
             self.assertEqual(action, sorted_actions[index])
 
+
+class CaseRebuildSimpleTest(SimpleTestCase):
+
+    def test_rebuild_with_opened_on_in_update(self):
+        case = CommCareCase(
+            actions=[
+                CommCareCaseAction.wrap({
+                    "xform_id": "8166142bceefae35375abfdf285f9dfc",
+                    "user_id": "66925160e15a21bc6f332e967dc05647",
+                    "server_date": "2013-02-07T04:40:40Z",
+                    "action_type": "update",
+                    "updated_known_properties": {
+                        "opened_on": "2012-10-18T12:16:22Z",
```

Comment:
This is quite likely the same issue (I see that the date in the reproducing
test starts with `2012-10`, and the error was also about there being a
unicode where a datetime was expected). I think revisiting in a couple
weeks is fine.

On Fri, Oct 2, 2015 at 2:05 AM Cory Zue <notifications@github.com> wrote:

> In corehq/ex-submodules/casexml/apps/case/tests/test_rebuild.py
> <https://github.com/dimagi/commcare-hq/pull/8416#discussion_r40993434>:
>
> > @@ -509,6 +509,30 @@ def _test(self, action_tuples):
> >          for index, action in action_tuples:
> >              self.assertEqual(action, sorted_actions[index])
> >
> > +
> > +class CaseRebuildSimpleTest(SimpleTestCase):
> > +
> > +    def test_rebuild_with_opened_on_in_update(self):
> > +        case = CommCareCase(
> > +            actions=[
> > +                CommCareCaseAction.wrap({
> > +                    "xform_id": "8166142bceefae35375abfdf285f9dfc",
> > +                    "user_id": "66925160e15a21bc6f332e967dc05647",
> > +                    "server_date": "2013-02-07T04:40:40Z",
> > +                    "action_type": "update",
> > +                    "updated_known_properties": {
> > +                        "opened_on": "2012-10-18T12:16:22Z",
>
> (i set my self a calendar reminder for two weeks from now to revert this
> PR if we don't see any issues)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/8416/files#r40993434>.
>